src/content/migrate/5.md

```diff
@@ -67,6 +67,7 @@ Update the following options to their new version (if used):
 - `NamedChunksPlugin` => `optimization.chunkIds: 'named'`
 - `HashedModulesPlugin` => `optimization.moduleIds: 'hashed'`
 - `optimization.occurrenceOrder: true` => `optimization: { chunkIds: 'total-size', moduleIds: 'size' }`
+- Renamed `optimization.splitChunks.cacheGroups.vendors` => `optimization.splitChunks.cacheGroups.defaultVendors`
```

Comment:
```suggestion
- `optimization.splitChunks.cacheGroups.vendors` => `optimization.splitChunks.cacheGroups.defaultVendors`
```
This looks more consistent to me.

Reply:
suggestion @montogeek 
I don't think thats a good option because for others it is update of values but here it is renamed

Reply:
@jeffin143  I think all corresponds to changes ( either rename or update ) that needs to be done by an user thus all having same notation makes more sense to me.

Reply:
Yeah arrow says it in the others dont see why we should add word `Renamed` here exclusively, just remove it and its good

Reply:
@jeffin143 I also agree with the suggestion by @rishabh3112.

Reply:
Either looks fine
but I would lean more towards having `rename`, its safe to have as its actually just renamed method and it might be helpful for other dev while migrating by assuring that -its the same old method but changed the name is changed, so it's not a breaking change-

Reply:
> This looks more consistent to me.

Thanks, made changes :)